<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>216</width>
      <height>268</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_id"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_year"/>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEdit_places"/>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="lineEdit_rNumber"/>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="lineEdit_model"/>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="lineEdit_vinCode"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_price"/>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="lineEdin_door"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_year">
         <property name="font">
          <font>
           <family>System</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>year</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_price">
         <property name="font">
          <font>
           <family>Sitka</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>price</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_places">
         <property name="font">
          <font>
           <family>Sitka</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>places</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_rNumber">
         <property name="font">
          <font>
           <family>System</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>rNumber</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_door">
         <property name="font">
          <font>
           <family>Sitka</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>door</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_model">
         <property name="font">
          <font>
           <family>Sitka</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>model</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_vinCode">
         <property name="font">
          <font>
           <family>Sitka</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>vinCode</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_id">
         <property name="font">
          <font>
           <family>System</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_add">
       <property name="text">
        <string>Add car</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionCreate"/>
    <addaction name="actionShow"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCreate">
   <property name="text">
    <string>Create </string>
   </property>
  </action>
  <action name="actionShow">
   <property name="text">
    <string>Show</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
